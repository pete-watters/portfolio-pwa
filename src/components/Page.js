import React from 'react';
import PropTypes from 'prop-types';
import { BrowserRouter } from 'react-router-dom';
import Header from './Header';
import Section from './Section';

const Page = ({ title, children }) =>
    <>
    <BrowserRouter>
      <Header />
    </BrowserRouter>
    <Section title={title}>
        <h2>{title}</h2>
        {children}
    </Section>
    </>;

Page.propTypes = {
  title: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
};

export default Page;