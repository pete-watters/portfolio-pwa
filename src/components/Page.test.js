import React from 'react';
import { shallow } from 'enzyme';
import Page from './Page';

describe('Page', () => {
  it('should render the component correctly', () => {
    expect(shallow(<Page />)).toMatchSnapshot();
  });
});
