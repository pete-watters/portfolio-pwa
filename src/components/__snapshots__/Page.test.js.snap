// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render the component correctly 1`] = `
<Memo(Section)>
  <article />
</Memo(Section)>
`;
